Tools/TestWebKitAPI/Tests/WTF/URLParser.cpp

```diff
@@ -944,8 +944,13 @@ TEST_F(WTF_URLParser, ParserDifferences)
     checkURL(utf16String(u"http://ԛәлп.com"), { "http"_s, ""_s, ""_s, "xn--k1ai47bhi.com"_s, 0, "/"_s, ""_s, ""_s, "http://xn--k1ai47bhi.com/"_s });
     checkURLDifferences(utf16String(u"http://Ⱥbby.com"),
         { "http"_s, ""_s, ""_s, "xn--bby-iy0b.com"_s, 0, "/"_s, ""_s, ""_s, "http://xn--bby-iy0b.com/"_s });
+#if PLATFORM(WIN) // After ICU 76.1
```

Comment:
Do you suggest to use `U_ICU_VERSION_MAJOR_NUM`? I revised this PR. Does it look good to you?

Reply:
This seems potentially problematic as the ICU changes have not been vetted and this would mean we don't see this "regression" when we update on the Apple side. cc @achristensen07

Reply:
This is not about displaying domain names, but whether a URL containing a Unicode domain parses or not. This currently does not parse and I guess soon it might? And there's probably other changes? I would be surprised if this doesn't hit web-platform-tests results, but Windows doesn't run those I guess.

https://jsdom.github.io/whatwg-url/#url=aHR0cHM6Ly/ihLIv&base=YWJvdXQ6Ymxhbms=